shotover-proxy/config/config.yaml

```diff
@@ -1,3 +1,3 @@
 ---
-main_log_level: "info,shotover_proxy=info"
+main_log_level: "info"
```

Comment:
I think the reason this is like this is like this is to demonstrate how to set separate log levels for shotover and everything else.
I think what we should do is add a comment above explaining how to configure the log level.
But that should go in a separate PR, so for now just revert this change.